### ISSUE TITLE
html-formatter: display a summary of the execution

### ISSUE THREAD

**Issue by vincent-psarga** [cucumber/cucumber#1056]
## Summary

It's a feedback from a `cucumber-ruby` 4 user that can be found here: https://community.smartbear.com/t5/Cucumber-Open/Cucumber-Ruby-4-0-0-has-been-released/m-p/203029/highlight/true#M32

The user misses the summary that was in the old HTMl report and I think it'd be really interesting to put that back in the HTML report.

**Comment by aslakhellesoy** [comment]
I agree. I think we should start with some mockups of how this summary would look:

- What does the summary widget look like?
- Where is it displayed?
- Is it interactive? If so, how?

Pull requests should be made agains `@cucumber/react` - https://github.com/cucumber/cucumber/tree/master/react
